qa.gemspec

```diff
@@ -22,10 +22,13 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.6"
   s.add_dependency "activerecord-import"
   s.add_dependency "deprecation"
+  s.add_dependency "rdf", "2.2.3"
```

Comment:
NOTE: 2.2.5 is not marked as the latest official release in rdf/rdf  2.2.3 is.  But it is what is returned by rubygems.org when you bundle install/update.